modules/turnos/controller/agenda.ts

```diff
@@ -878,6 +881,53 @@ export function actualizarTurnosDelDia() {
     });
 }
 
+/**
+ * El dia anterior a la ejecución de la agenda (a las 12 del mediodía), los turnos restantes mobile se setean a 0
+ *
+ * @export actualizarTurnosMobile()
+ * @returns resultado
+ */
+export async function actualizarTurnosMobile() {
+    const fechaActualizar = moment().add(1, 'day');
+
+    const condicion = {
+        $or: [{ estado: 'disponible' }, { estado: 'publicada' }],
+        horaInicio: {
+            $gte: (moment(fechaActualizar).startOf('day').toDate() as any),
+            $lte: (moment(fechaActualizar).endOf('day').toDate() as any)
+        },
+        'bloques.restantesMobile': { $gt: 0 }
+    };
+    const cursor = agendaModel.find(condicion).cursor();
+    let logRequest = {
+        user: {
+            usuario: { nombre: 'actualizarTurnosMobileJob', apellido: 'actualizarTurnosMobileJob' },
+            app: 'citas',
+            organizacion: userScheduler.user.organizacion.nombre
+        },
+        ip: 'localhost',
+        connection: {
+            localAddress: ''
+        }
+    };
+    try {
+        return await cursor.eachAsync(async doc => {
+            const agenda: any = doc;
+            for (let j = 0; j < agenda.bloques.length; j++) {
+                if (agenda.bloques[j].restantesMobile > 0) {
+                    agenda.bloques[j].restantesMobile = 0;
+                }
+            }
+            Auth.audit(agenda, (userScheduler as any));
+            await saveAgenda(agenda);
+            await log(logRequest, logKeys.turnosMobileUpdate.key, null, logKeys.turnosMobileUpdate.operacion, null, { idAgenda: agenda._id });
+        });
+    } catch (err) {
+        await log(logRequest, logKeys.turnosMobileUpdate.key, null, logKeys.turnosMobileUpdate.operacion, err, null);
+        return Promise.reject(err);
```

Comment:
No es necesario el Promise.reject
```suggestion
        return Promise.reject(err);
```

Reply:
cambios realizados